docs/reference/aggregations/bucket/time-series-aggregation.asciidoc

```diff
@@ -0,0 +1,100 @@
+[[search-aggregations-bucket-time-series-aggregation]]
+=== Time Series Aggregation
+++++
+<titleabbrev>Time Series</titleabbrev>
+++++
+
+The time series aggregation queries data created using a time series index. This is typically data such as metrics
```

Comment:
I think `preview::[]` should be added here to indicate that this feature is techpreview.